tests/lib/rules/no-deprecated.js

```diff
@@ -39,151 +47,316 @@ ruleTester.run('no-deprecated', rule, {
+    {
+      code: 'var {createClass, PropTypes} = require(\'react\');',
+      parser: 'babel-eslint',
+      errors: [{
+        message: 'React.createClass is deprecated since React 15.5.0, use the npm module create-react-class instead'
+      }, {
+        message: 'React.PropTypes is deprecated since React 15.5.0, use the npm module prop-types instead'
+      }]
+    },
+    {
+      code: 'import {createClass} from \'react\';',
+      parser: 'babel-eslint',
+      errors: [{
+        message: 'React.createClass is deprecated since React 15.5.0, use the npm module create-react-class instead'
+      }]
+    },
+    {
+      code: 'import {createClass, PropTypes} from \'react\';',
+      parser: 'babel-eslint',
+      errors: [{
+        message: 'React.createClass is deprecated since React 15.5.0, use the npm module create-react-class instead'
+      }, {
+        message: 'React.PropTypes is deprecated since React 15.5.0, use the npm module prop-types instead'
+      }]
+    },
+    {
+      code: `
       import React from 'react';
       const {createClass, PropTypes} = React;
     `,
-    parser: 'babel-eslint',
-    errors: [{
-      message: 'React.createClass is deprecated since React 15.5.0, use the npm module create-react-class instead'
-    }, {
-      message: 'React.PropTypes is deprecated since React 15.5.0, use the npm module prop-types instead'
-    }]
-  }, {
-    code: 'import {printDOM} from \'react-addons-perf\';',
-    parser: 'babel-eslint',
-    errors: [{
-      message: 'ReactPerf.printDOM is deprecated since React 15.0.0, use ReactPerf.printOperations instead'
-    }]
-  }, {
-    code: `
+      parser: 'babel-eslint',
+      errors: [{
+        message: 'React.createClass is deprecated since React 15.5.0, use the npm module create-react-class instead'
+      }, {
+        message: 'React.PropTypes is deprecated since React 15.5.0, use the npm module prop-types instead'
+      }]
+    },
+    {
+      code: 'import {printDOM} from \'react-addons-perf\';',
+      parser: 'babel-eslint',
+      errors: [{
+        message: 'ReactPerf.printDOM is deprecated since React 15.0.0, use ReactPerf.printOperations instead'
+      }]
+    },
+    {
+      code: `
       import ReactPerf from 'react-addons-perf';
       const {printDOM} = ReactPerf;
     `,
-    parser: 'babel-eslint',
-    errors: [{
-      message: 'ReactPerf.printDOM is deprecated since React 15.0.0, use ReactPerf.printOperations instead'
-    }]
-  },
-  {
-    code: 'React.DOM.div',
-    errors: [{
-      message: 'React.DOM is deprecated since React 15.6.0, use the npm module react-dom-factories instead'
-    }]
-  }]
+      parser: 'babel-eslint',
+      errors: [{
+        message: 'ReactPerf.printDOM is deprecated since React 15.0.0, use ReactPerf.printOperations instead'
+      }]
+    },
+    {
+      code: 'React.DOM.div',
+      errors: [{
+        message: 'React.DOM is deprecated since React 15.6.0, use the npm module react-dom-factories instead'
```

Comment:
can all these tests be migrated to use the `errorMessage` helper as well?

Reply:
sure thing, see the updated tests